recipes/clmm/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "clmm" %}
+{% set version = "1.14.40" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/clmm-{{ version }}.tar.gz
+  sha256: edfab4226e7eba3a21b5cf938d5ac1a72cedfa020c6c6d9796e1021a99b13afc
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<=3.12
```

Comment:
Please have a look at https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python with regards to using the `python_min` variable for `noarch: python` packages. Mostly you'll have to change the `python` listed under `host`, `run` and then add it in `test`/`requires`.